src/main/resources/fabric.mod.json

```diff
@@ -27,9 +27,7 @@
   ],
 
   "depends": {
-    "fabricloader": ">=0.7.4"
-  },
-  "suggests": {
```

Comment:
why remove the suggests block?

Reply:
because I often see mods which leave this block unchanged in there code

Reply:
That's the fault of the mod author, though. I don't think we should remove it because this is the example mod.json